src/commands/metrics.ts

```diff
@@ -0,0 +1,42 @@
+import { Command } from '@oclif/core'
+import { Jamsocket } from '../jamsocket'
+
+type BackendMetrics = {
+  cluster: string,
+  backend_id: string,
+  mem_used: number,
+  mem_available: number,
+  cpu_used: number,
+  sys_cpu: number
+}
+
+const formatToHeader = (a: string, header: string) => a.padEnd(header.length, ' ')
+export default class Metrics extends Command {
+  static description = 'Stream metrics from a running backend'
+  static examples = [
+    '<%= config.bin %> <%= command.id %> f7em2',
+  ]
+
+  static args = [
+    { name: 'backend', description: 'The name of the backend, a random string of letters and numbers returned by the spawn command.', required: true },
+  ]
+
+  public async run(): Promise<void> {
+    const jamsocket = Jamsocket.fromEnvironment()
+    const { args } = await this.parse(Metrics)
+    const boldStart = '\u001B[1m'
+    const boldEnd = '\u001B[0m'
+    this.log(boldStart + 'cpu_util\tmem_used\tmem_avail' + boldEnd)
```

Comment:
We're using `chalk` elsewhere for formatting text. Should be easy enough:

```ts
this.log(chalk.bold`cpu util\tmem used\tmem avail`)
```

(Also I'd drop the underscores in the headers.)